llm_assistant/discord_bot/discord_bot.py

```diff
@@ -42,18 +49,22 @@ async def my_long_command_function(ctx: interactions.SlashContext):
   name="prompt", description="Enter your prompt", required=True, opt_type=interactions.OptionType.STRING, min_length=10
 )
 async def ask_model(ctx: interactions.SlashContext, model: str = "", prompt: str = ""):
-  await ctx.send(f'You asked model {model} with the prompt: "{prompt}"')
+  if model not in MODEL_CHOICES:
+    await ctx.send(f"Invalid model `{model}`. Please choose from `{MODEL_CHOICES}`.")
+    return
+  await ctx.defer()
+
+  response, response_time = await llm_assistant.discord_bot.qa.answer_question(model, prompt, AI_SERVER_URL)
+  await ctx.send(response)
 
 
 @ask_model.autocomplete("model")
 async def autocomplete(ctx: interactions.AutocompleteContext):
-  string_option_input = ctx.input_text  # note: can be empty
-  print(f"input: {string_option_input}")
+  string_option_input = ctx.input_text  # note: result can be empty
```

Comment:
Instead of result can be blank, can we do either?

- Make this a required option, so we know its always there?
- Or, make a default option if we're given blank?
- Or, make this an autocomplete so the user can choose which model to use?

Reply:
I think we just forgot to uncomment this because it is an autocomplete with required flag on

```python
@interactions.slash_option(
  name="model",
  description="Choose an LLM model",
  required=True,
  opt_type=interactions.OptionType.STRING,
  autocomplete=True,
)
```